helpers/config.js

```diff
@@ -90,22 +91,22 @@ function Config(packageJson, parseCommandLineOptions = true) {
 	};
 
 	let commandLineConfig = {
-		wsPort: +program.port || process.env.LISK_WS_PORT || null,
-		httpPort: +program.httpPort || process.env.LISK_HTTP_PORT || null,
-		address: program.address || process.env.LISK_ADDRESS || null,
-		fileLogLevel: program.log || process.env.LISK_FILE_LOG_LEVEL || null,
-		consoleLogLevel: process.env.LISK_CONSOLE_LOG_LEVEL || null,
-		cacheEnabled: process.env.LISK_CACHE_ENABLED === 'true',
+		wsPort: +program.port || getenv(process.env.LISK_WS_PORT),
```

Comment:
It should be wsPort instead of port.

Reply:
No this is `port` came from this line https://github.com/LiskHQ/lisk/blob/71ab88cdfaf06b71038e3a88644ebab5d2b1441d/helpers/config.js#L50

Didn't change command parameters for backward compatibility.